### ISSUE TITLE
`tx_info` endpoint sometimes returns random assets in the `collateral_output` field

### ISSUE THREAD

**Issue by fallen-icarus** [cardano-community/koios-artifacts#284]
**Describe the bug**
Consider these two transactions:
```bash
168c1c4b71595c6e2c4191cea58dc58667657f04afb677bc70e0ac4b7da9f7bc
6fc9e482959befc6928d2691caa62c8a52a5abce167bd5fce8874cc315bb346d
```

In both transactions, the same UTxO is used for collateral:
```bash
4cc5755712fee56feabad637acf741bc8c36dda5f3d6695ac6487a77c4a92d76#0
```

The UTxO does *not* contain any native assets. However, querying the transactions with this query (one at a time):
```bash
curl -X POST "https://preprod.koios.rest/api/v1/tx_info?select=collateral_output,collateral_inputs"  -H "accept: application/json" -H "content-type: application/json"  -d '{"_tx_hashes":["6fc9e482959befc6928d2691caa62c8a52a5abce167bd5fce8874cc315bb346d"]}'
```
sometimes returns a collateral output with native assets. The native assets returned are not part of the transaction, and are not even in the addresses involved.

**Comment by rdlrt** [comment]
Thanks for raising @fallen-icarus , will be addressed via linked PR

**Comment by fallen-icarus** [comment]
Is the response now supposed to return a string when the collateral output's asset list is empty? The following query now occassionally returns `"[]"` instead of `[]`for the asset list:

```bash
curl -X POST "https://preprod.koios.rest/api/v1/tx_info?select=collateral_output->asset_list"  -H "accept: application/json" -H "content-type: application/json"  -d '{"_tx_hashes":["6fc9e482959befc6928d2691caa62c8a52a5abce167bd5fce8874cc315bb346d"]}'
```
The JSON response alternates between:
```json
[
  {
    "asset_list": "[]"
  }
]
```
and
```json
[
  {
    "asset_list": []
  }
]
```

**Comment by rdlrt** [comment]
Should be resolved as of Koios v1.2.0a , please report if you still see issues
